squad/settings.py

```diff
@@ -94,6 +95,7 @@
     'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'simple_history.middleware.HistoryRequestMiddleware',
```

Comment:
where is this being used?

Reply:
There are some templates for Django's admin area that simple-history uses, thus requiring it to be listed in INSTALLED_APPS: https://github.com/treyhunner/django-simple-history/tree/master/simple_history/templates/simple_history.

The middleware was to enable simple-history to keep record of which user changed the template. But I looked further in their code and that is being done already in SimpleHistoryAdmin. I'm removing this line.

Reply:
The middleware was to enable simple-history to keep record of which user changed the template. But I looked further in their code and that is being done already in SimpleHistoryAdmin. I'm removing this line.